tests/webapp/api/test_performance_alertsummary_api.py

```diff
@@ -89,6 +89,7 @@ def test_alert_summaries_get(client, test_perf_alert_summary, test_perf_alert):
             'revision',
```

Comment:
We need to add some end-to-end backend tests. To ensure that the user can: 
* add multiple tags to a summary
* remove a tag from a summary
* not add an unregistered perf tag to a summary